frontend/src/components/Testing/Test.jsx

```diff
@@ -155,13 +156,20 @@ const Test = ({
           test_disabled: dataSendInProcess,
         })}
       >
-        <button
-          className={classNames(
-            { "test__start-sign": dataSendInProcess === false },
-            { "test__start-sign_hidden": dataSendInProcess === true }
-          )}
-          onClick={onPlayClick}
-        />
+        <HoverTip
+          render={() =>
+            finalText === "Все тесты" ? "Запуск тестов" : "Запуск теста"
```

Comment:
```suggestion
            id === -1 ? "Запуск тестов" : "Запуск теста"
```
все таки мож с текстом не будем сравнивать? )

Reply:
Согласен) 3621f7499d12e3e3732bdbedc71122acaab34e06